src/applications/vaos/utils/selectors.js

```diff
@@ -165,6 +166,17 @@ export function getChosenClinicInfo(state) {
   );
 }
 
+export function getReasonForAppointment(state, pageKey) {
+  const formInfo = getFormPageInfo(state, pageKey);
+  const remainingChar = getNewAppointment(state).reasonRemainingChar;
+  return {
+    ...formInfo,
+    reasonRemainingChar: !isNaN(remainingChar)
```

Comment:
In which situation would `remainingChar` be `NaN`? Does it just need to have a default value set in the reducer?